package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "codejam_keyboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": ".eslintrc.js",
+  "dependencies": {
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-import": "^2.18.2"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wlukla/virtual-keyboard.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/wlukla/virtual-keyboard/issues"
+  },
+  "homepage": "https://github.com/wlukla/virtual-keyboard#readme"
+}
```

Comment:
It is better to add .gitignore file to the project repository. This way, node_modules directory , for example, can be excluded from tracking by git.